docs/examples/code-generation.ipynb

```diff
@@ -0,0 +1,817 @@
+{
```

Comment:
<p>An interesting alternative could be telling it to ask questions as needed, such as what kind of authentication to use, etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/datastax/graph-rag/pull/164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>